sdk/storage/Azure.Storage.Blobs/tests/EncryptedBlockBlobClientTests.cs

```diff
@@ -18,25 +25,281 @@ public EncryptedBlockBlobClientTests(bool async)
         {
         }
 
-        // Placeholder test to verify things work end-to-end
-        [Test]
-        public async Task DeleteAsync()
+        #region Utility
+
+        /// <summary>
+        /// Gets a client to a nonexistent blob using client-side encryption in a brand new disposable container.
+        /// </summary>
+        /// <param name="blob">The blob client created.</param>
+        /// <returns>The IDisposable to delete the container when finished.</returns>
+        private IDisposable GetEncryptedBlockBlobClient(out EncryptedBlobClient blob, MockKeyEncryptionKey mock)
+        {
+            var disposable = this.GetNewContainer(out var container);
+            blob = new EncryptedBlobClient(
+                    new Uri(Path.Combine(container.Uri.ToString(), this.GetNewBlobName())), this.GetNewSharedKeyCredentials(),
```

Comment:
Nit - use `BlobUriBuilder` or even `container.GetBlobClient(GetNewBlobName()).Uri`.

Reply:
Nit - put each parameter on its own line

Reply:
Ideally we'd create the `BlobServiceClient` using these credentials so `GetNewContainer` and `GetNewSharedKeyCredentials` never get out of alignment.

Reply:
I don't understand this last comment.